### ISSUE TITLE
Throw missing await error on expect.poll

### ISSUE THREAD

**Issue by moufmouf** [playwright-community/eslint-plugin-playwright#97]
Hey!

First of all, thanks for this library, it saved my life when I was desperate to find an issue with a Playwright test. Good job, really.

Here is a proposal for a possible improvement:

The `expect.poll` method can be used to query arbitrary data (not necessarily locators) and in this case, any method called after `expect` should probably raise a warning.

For instance:

```js
expect.poll(() => { /* some code */ }).toContain('foo');
```

should probably raise an eslint error about a missing `await`.

Right now, it is not raising an error, because `toContain` is not in the list of checked methods for `expect`.
